src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -70,12 +70,14 @@ export const LargeArtistHeader: SFC<Props> = props => {
         data={carousel.images as any}
         render={slide => {
           return (
-            <Image
-              px={5}
-              src={slide.resized.url}
-              width={slide.resized.width}
-              height={slide.resized.height}
-            />
+            <a href={slide.href}>
```

Comment:
Nice, I had to doublecheck what `href` in the context of an image meant...turns out for the carousel we set it to the show or artwork page: https://github.com/artsy/metaphysics/blob/4257e868be805ddd801629b894d87f2e83e161f0/src/schema/artist/carousel.js#L51

Sweet!

Reply:
Yeah! Thanks for your tip earlier about the links being named `href`, that's what tipped me off to it, heh.